tests/AtlasTest.php

```diff
@@ -728,41 +728,8 @@ public function testCalcPrimary()
                 ],
             ],
             'taggings' => [
-                0 => [
```

Comment:
Is that a better solution? I figured that given the overhead in setup and the fact that there are 3 other articles, it made sense to just have a single one of the threads have no tags and the other two still have them.